android/rctmgl/src/main/java/com/mapbox/rctmgl/location/LocationManager.java

```diff
@@ -110,6 +110,7 @@ public void disable() {
         if (locationEngine == null) {
             return;
         }
+        locationEngine.removeLocationUpdates();
```

Comment:
as per the documentation, you need to call ```removeLocationUpdates()``` before you deactivate.

Reply:
👍 great catch